CMS-Enterprise/sbom-harbor
# refactor: organize terraform into modules

## Summary

This change takes all the raw terraform config imported under `generated/` and refactors into dynamic deployable config under `infrastructure/`

### Added

none

### Changed

Nothing functional was changed.

### Deprecated

none

### Removed

none

### Fixed

none

## How to test

With Terraform install, you can 
```bash
cd infrastructure
terraform init
ENVIRONMENT=sandbox terraform plan
```

Sandbox environments are the easiest to test with at this time since they should be clean of any sbom-harbor related resources.

You should see a plan describing all the resources configured under `infrastructure/`
